src/main/java/org/idpass/lite/Card.java

```diff
@@ -47,21 +38,68 @@
     private boolean isAuthenticated = false;
     private byte[] cardPublicKey = null;
     private byte[] cardAsByte = null;
+    private int scale = 1;  // scale of 1 is best for unit tests
```

Comment:
Why do we optimize for unit test and not for normal usage?

Reply:
- `app` is the default folder name when I created it
- layout.xml is now removed

A scale of 1 corresponds to 1 pixel per module and runs perfectly without intermittent error during unit testes. Scaling it up will result in very intermittent false alert unit test error (1 out of 50).  I tried to research more and found the explanation [here](https://github.com/journeyapps/zxing-android-embedded/issues/395)